README.md

```diff
@@ -34,10 +34,10 @@ None
 
 ### Configuration
 
-| ENV field | Req. / Opt.  | Description                                                |
-|-----------|--------------|------------------------------------------------------------|
-| INTERVAL  | *Optional*   | Interval of pruning, default is `86400` seconds.           |
-| OBJECTS   | *Optional*   | Objects to be pruned, default is `container volume image`. |
+| ENV field | Req. / Opt.  | Description                                                        |
+|-----------|--------------|--------------------------------------------------------------------|
+| INTERVAL  | *Optional*   | Interval of pruning, default is `86400` seconds (24h).             |
+| OBJECTS   | *Optional*   | Objects to be pruned, default is `container volume image network`. |
```

Comment:
This is not correct, as the default in the Dockerfile is still `container volume image`, see [here](https://github.com/jokay/docker-prune/blob/main/src/Dockerfile#L4).

In general it's a good idea to add `network` as example / option param here but the description need to be changed to reflect the default and the additional option.

There may be setups where an (external) network is created for multiple services, e.g. proxy network once (or manually) and a prune would remove them as well. This is why I decided to have `network` not pruned as default :wink:

Reply:
Will merge it and change this afterwards 👍🏼